yoti_python_sdk/tests/doc_scan/session/create/test_sdk_config.py

```diff
@@ -41,12 +43,23 @@ def test_should_build_correctly(self):
         assert result.success_url is self.SOME_SUCCESS_URL
         assert result.error_url is self.SOME_ERROR_URL
         assert result.privacy_policy_url is self.SOME_PRIVACY_POLICY_URL
+        # assert result.allow_handoff is self.SOME_ALLOW_HANDOFF
```

Comment:
done

Reply:
done

Reply:
It looks like you've removed this check, rather than uncommenting the line. Was that intentional? 

Please could you try not to force push too when possible? It makes it harder to check whether a new commit has addressed any issues, and makes all the comments appear "Outdated", even if they might not be

Reply:
ok, I will stop force push

Reply:
Thanks, and what about my first question:
> It looks like you've removed this check, rather than uncommenting the line. Was that intentional?

Reply:
for me it's looks like I uncommented code, done